splinterd/src/main.rs

```diff
@@ -443,25 +446,52 @@ fn main() {
 
     let matches = app.get_matches();
 
-    let log_level = match matches.occurrences_of("verbose") {
-        0 => log::LevelFilter::Warn,
-        1 => log::LevelFilter::Info,
-        2 => log::LevelFilter::Debug,
-        _ => log::LevelFilter::Trace,
-    };
-
-    let mut log_spec_builder = LogSpecBuilder::new();
-    log_spec_builder.default(log_level);
-    log_spec_builder.module("hyper", log::LevelFilter::Warn);
-    log_spec_builder.module("tokio", log::LevelFilter::Warn);
-    #[cfg(feature = "https-bind")]
-    log_spec_builder.module("h2", log::LevelFilter::Warn);
-
-    Logger::with(log_spec_builder.build())
-        .format(log_format)
-        .log_target(flexi_logger::LogTarget::StdOut)
-        .start()
-        .expect("Failed to create logger");
+    #[cfg(feature = "log-config")]
```

Comment:
This doesn't seem to take clap's log level into account. Similarly, hyper, tokio, h2 log-levels aren't adjusted.

Reply:
fixed

Reply:
fixed/refactored